hydra-node/hydra-node.cabal

```diff
@@ -176,6 +176,7 @@ executable mock-chain
 
 test-suite tests
   import:             project-config
+  ghc-options:        -threaded -rtsopts -with-rtsopts=-N2
```

Comment:
Proposal: Use `-maxN4` to reap out more parallelism if available

Reply:
Isn't `-N` without `2` the default and uses as many cores as available? (unless overridden when calling)

Reply:
With `-N`, my CPU goes above 900% usage and my fans get crazy for a sec. Plus, because most of our tests are quite short, we don't get much benefits from _too much parallelism_ as the scheduler spend more time managing threads than doing actual work :sweat_smile: ... 

I've tried some parameters, and seems like 2 is nice enough, provides speed up but keeps the laptop cool and works against climate change :fire:

Reply:
As you are running tests through cabal, there might be a way to have per-user configuration. I for my part would like to benefit from the 24 threads of my cpu 😅